docs/testplan/pfc/PFC_PAUSE_RESPONSE_HEADROOM_README.md

```diff
@@ -0,0 +1,48 @@
+
+## Testbed Setup
+The testbed consists of two IXIA ports and a SONiC device under test (DUT) as follows. Both IXIA ports should have the same bandwidth capacity. To reduce the configuration complexity, we recommond configuring the switch as a Top of Rack (ToR) / Tier 0 (T0) switch and binding two switch interfaces to the Vlan.
+
+```
+                     _________
+                    |         |
+IXIA tx port ------ |   DUT   |------ IXIA rx port
+                    |_________|
+```
+In addition, [PFC watchdog](https://github.com/sonic-net/SONiC/wiki/PFC-Watchdog-Design) must be disabled at the SONiC DUT. Otherwise, the DUT will trigger PFC watchdog to drop packets when it detects persistent PFC pause storms. The command to disable PFC watchdog is <code>sudo pfcwd stop</code>.
+
+## Experiment Steps
+In this experiment, we need to create three traffic items:
+
+- Test data traffic: Data packets sent from the IXIA tx port to the IXIA rx port at the lossless priorities (e.g., e.g., 3, 4 or both 3 and 4) that we want to test. Note that the packets should be marked with the correct DSCP value (e.g., DSCP 3 for priority 3). The traffic demand should be 50% of the line rate.
+
+- Background data traffic: Data packets sent from the IXIA tx port to the IXIA rx port at the other priorities that we want to test (e.g., 0-2 and 5-7). The traffic demand should be 50% of the line rate.
+
+- PFC pause storm: Persistent PFC pause frames from the IXIA rx port to the IXIA tx port. The priorities of PFC pause frames should be same as those of test data traffic. And the inter-frame transmission interval should be smaller than per-frame pause duration.
+
+This experiment needs the following five steps:
+
+- Start test data traffic and background data traffic simultaneously.
+
+- After a fixed duration (e.g., 3 second), start the PFC pause storm to fully block the test priorities at the switch.
+
+- Stop PFC pause storm after a fixed duration (eg. 10 second)
+
+- Immediately after (eg. + 2 seconds) stopping the pause storm, stop test data traffic and background data traffic.
+
+- Check if the IXIA rx port receives (1) all the sent frames of background data traffic (which should not be impacted by PFC), and (2) non-zero number of frames of test data traffic (PFC pause storm was triggered after traffic flow was started). Check if (3) the total number of transmitted bytes of test data traffic is smaller than the switch shared buffer size for certain pfc response times, and greater for other response times since the headroom limit would be exceeded due to the high PFC response delay.
```

Comment:
I think you should discuss test results based on how much PFC delay you add (or whether you add PFC delay).